fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -541,7 +630,7 @@ private fun TopDrawer(
                 drawerConstraints
                     .offset { IntOffset(0, 0) }
                     .semantics {
-                        if (drawerState.isOpen) {
+                        if (drawerState.hasOpenedState && !drawerState.isClosed) {
```

Comment:
not required "drawerState.hasOpenedState"

Reply:
okay, removed "drawerState.hasOpenedState"